tools/session-ui/public/js/load.js

```diff
@@ -0,0 +1,251 @@
+/* When the user clicks on the button,
+toggle between hiding and showing the dropdown content */
+
+function filterFunction() {
+  var input, filter, ul, li, a, i;
+  input = document.getElementById("Search");
+  filter = input.value.toLocaleString();
+  div = document.getElementById("menu1");
+  a = div.getElementsByTagName("a");
+  for (i = 0; i < a.length; i++) {
+    if (a[i].innerHTML.toLowerCase().indexOf(filter) > -1) {
+       a[i].style.display = "";
+    } else {
+      a[i].style.display = "none";
+    }
+  }
+}
+
+
+function modal(){
+    //alert("Dhawan was here");
+
+    var xhr0=new XMLHttpRequest();
+    var usl0="http://127.0.0.1:3000/modal"
+    data=document.getElementById("sidebar").innerText
+    xhr0.open("POST",url);
+    xhr0.send()
+}
+
+$(document).ready(function(){
+   $("#sidebar").submit(function(){
+       $.ajax({
+           url: "http://127.0.0.1:3000/modal",
+           data: $("#modal").innerText,
+           success: function (data) {
+
+           }
+       })
+   })
+});
+
+
+function postModule() {
+    var xhr = new XMLHttpRequest();
+    var url = "http://127.0.0.1:3000/post";
```

Comment:
The `url` values are hard coded full paths, make them relative.

`/post`

This will allow the browser to complete the request correctly and make the code more portable binding to other addresses or ports.